core/src/test/java/org/elasticsearch/search/suggest/phrase/DirectCandidateGeneratorTests.java

```diff
@@ -113,12 +114,13 @@ public void testFromXContent() throws IOException {
         for (int runs = 0; runs < NUMBER_OF_RUNS; runs++) {
             DirectCandidateGeneratorBuilder generator = randomCandidateGenerator();
             XContentBuilder builder = XContentFactory.contentBuilder(randomFrom(XContentType.values()));
+            generator.toXContent(builder, ToXContent.EMPTY_PARAMS);
+            XContentBuilder shuffled = shuffleXContent(builder, Collections.emptySet());
```

Comment:
Same deal here with prettyPrint. Also, why not `generator = shuffleXContent(builder, emptySet)`? Should that `Set` parameter actually be `String...` I wonder?